tests/filesystem.go

```diff
@@ -405,16 +405,6 @@ func intJoin(ints []int, delimiter string) string {
 	return strings.Join(strArr, delimiter)
 }
 
-func removeEmpty(strings []string) []string {
```

Comment:
Unused since it was added in 654289f49d2d35309699bbae860dd0017408aae3, probably due to CR refactoring.